share/mk/suite.test.mk

```diff
@@ -118,8 +118,7 @@ beforecheck:
 #       times. "aftercheck" won't be run if "make check" fails, is interrupted,
 #       etc.
```

Comment:
It's not clear to me if the "this is intentional" line means that it's intended to set flags after `clean`, and letting these errors through. That seems strange to me.

I addressed this because when running the tests, it was unclear to me if something was wrong with my setup. Error messages shouldn't come up as a part of a normal test run, unless they're expected.

Also: I don't know if this is the sort of thing I should make a Problem Report first for? So a Bugzilla PR "make check gives noisy operation not permitted errors" so there's a place to track / discuss it, and then a github PR to fix it?

Seemed small enough to me that I could just make a GH PR.

Reply:
The commit message in 38f8fddf054fd has some backstory. I've added @ngie-eign as a reviewer.